shared/js/background/redirect.es6.js

```diff
@@ -93,7 +92,7 @@ function handleRequest (requestData) {
          * If request is a tracker, cancel the request
          */
 
-        let tracker = trackers.getTrackerData(requestData.url, thisTab.site.url, requestData)
+        let tracker = trackerutils.getTrackerData(requestData.url, thisTab.site.url, requestData)
```

Comment:
Why don't we make this change upstream, or extend from `Trackers` in `trackers.es6.js`? I think there is a lot of cases where we are using `trackers` directly, would we have to update every one to `trackerutils`?

Reply:
Right, I was trying not to modify this code given the short timeline however I think we should as you mention do it upstream and propagate it back.